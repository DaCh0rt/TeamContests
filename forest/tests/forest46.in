193 91
718 6166
-6307 8919
-3612 6279
-4849 3139
-8727 -787
7269 2225
-6215 6637
5401 -7384
-8895 -6445
-5898 2453
2400 7907
-7562 6594
-6818 7962
-3057 9999
7676 6890
8570 -713
-4042 -7973
-7641 8399
6034 9046
6246 1040
-1721 3070
-8636 8328
3018 8278
7760 -1323
6744 -4228
1206 -3649
3911 6463
-143 8918
-9808 -1965
-4367 5162
7154 3110
1346 3994
5571 -5228
-1719 9824
8290 -9978
6712 442
-554 -8374
3558 -72
2976 6130
-2783 5550
-1291 3974
-8947 -8457
1248 -9635
-1427 78
-7160 -7051
-5308 4616
-8625 -6675
9689 6778
-8781 70
8535 6808
1616 9786
7470 6278
2067 -2389
1017 -4830
7320 6961
-974 2811
-6423 -2779
441 -4601
2789 -3463
-5101 7862
-4223 2886
9956 -5927
-8275 9637
294 5878
3894 -8833
2161 2346
2436 -8412
2127 -8074
9476 7977
-9302 -6346
3019 -200
-1617 3291
-274 2933
-4245 -5373
-9499 19
-5510 3884
1649 6688
8070 -5980
-4520 600
-827 1087
-784 -4836
-4672 -3082
-8653 -4656
-2315 -9003
-6219 5654
6698 1528
4769 4900
8461 932
3910 -7833
1750 7708
-5175 -1160
688 -433
-8131 -8545
3961 -3797
-464 3859
5704 -8868
-2725 -6731
-3199 4290
-6394 9905
7093 -1810
6459 5419
46 -3303
-4839 -1256
8063 -3017
2339 5551
-7821 -4873
-8387 -5290
9942 5872
-62 -7731
5810 -6456
5520 -4957
7068 2047
3734 -9702
-5682 -3914
7400 -2436
-948 -6593
-8425 5352
-3079 4556
-6766 -2517
-4291 -7152
6252 3849
-2694 6517
-2717 1843
-2087 -7734
-9193 7725
1451 -5852
-5154 6633
-3529 643
8748 -9145
-7705 6994
-8353 8243
-6418 -7588
-2867 -6973
2426 2586
-3696 -7720
3739 3247
-6246 973
-8815 -323
-2782 55
-430 -7620
5594 -9179
4378 -4373
-2671 1655
-1565 8413
1863 1486
-8020 9650
3394 -6646
7235 570
-5888 -508
6022 -1001
-5097 6448
5347 7144
1870 -8813
-8852 -6930
-5859 7639
7293 5513
7524 8777
-4948 -8921
-4769 8723
-9190 8234
8375 -4922
5242 -6120
-1336 1542
-255 -5647
-8584 446
-1471 -3144
5900 -8590
7389 6450
3158 9429
6525 -9593
8678 6094
8820 1619
9570 8475
627 7592
-2660 -1664
-7763 -3241
5498 -5532
8011 159
-6643 1701
714 -4428
8981 4808
5309 949
7034 -3320
7405 -6867
-7850 5732
1384 5677
9108 9843
-7455 -1261
-8185 4043
-5356 -9037
2111 9046
9269 -7437
9154 -4730
