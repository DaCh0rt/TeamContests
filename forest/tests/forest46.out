73998.70 376334289.59
